cpp/basix/finite-element.h

```diff
@@ -725,24 +754,30 @@ class FiniteElement
            std::vector<std::tuple<std::vector<std::size_t>, std::vector<double>,
                                   xt::xtensor<double, 2>>>>
       _etrans_invT;
+
+  /// Indicates whether or not this is the discontinuous version of the element
+  bool _discontinuous;
 };
 
 /// Create an element using a given lattice type
 /// @param[in] family The element family
 /// @param[in] cell The reference cell type that the element is defined on
 /// @param[in] degree The degree of the element
 /// @param[in] lattice_type The lattice type that should be used to arrange DOF
-/// points of the element
-FiniteElement
-create_element(element::family family, cell::type cell, int degree,
-               lattice::type lattice_type);
+/// @param[in] discontinuous Indicates whether the element is discontinuous
```

Comment:
Could we add a brief comment on what different between continuous and discontinuous is in practice? E.g. comment in the dof associations.